truffle.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
   networks: {
     development: {
       host: ETH_HOST || "127.0.0.1",
-      port: ETH_PORT || 8545,
+      port: ETH_PORT || 7545,
```

Comment:
If you change this, change it also in port mapping of docker-compose.yml for ganache